### DISCUSSION TITLE
Processing incoming dmarc reports

### DISCUSSION THREAD

**Discussion by ra1u** [docker-mailserver/docker-mailserver#3717] [Q&A]
I enabled dmrac (because it seems fancy) in my mail server and now i get reports in mailbox (zipped xml reports).

Is there a tool so that i can automatically process received dmarc reports and have overview on what is going on?

**Comment by casperklein** [comment]
I did the same two years ago. But unfortunately I didn't find a useful tool/app back then.

Let me know, if you find something 😃

**Comment by MarcS1975** [comment]
There is [parsedmarc ](https://domainaware.github.io/parsedmarc/#) . I haven't tried it yet though.
